bip-0003.md

```diff
@@ -555,6 +543,7 @@ mentioned in the [Changelog](#changelog) section.
 - The "Post-History" header is replaced with the "Discussion" header.
 - The optional "Version" header is introduced.
 - The "Discussions-To" header is dropped as it has never been used in any BIP.
+- The "License-Code" header has been sunset, as it was used by only five BIPs (98, 116, 117, 330, 340) and created more ambiguity than clarity.
```

Comment:
Just noticed that this says License-Code was used by five BIPs, but below line 734, it says that License-Code was used four times. I must have overlooked BIP 340. But since line 546 is just being introduced here, perhaps it could be made consistent.

Reply:
Good catch, done.